frontend/javascripts/dashboard/dataset_view.js

```diff
@@ -187,6 +196,65 @@ function DatasetView(props: Props) {
     );
   }
 
+  function renderNewJobsAlert() {
+    const stateToIcon = {
+      UNKNOWN: <QuestionCircleTwoTone twoToneColor="grey" />,
+      PENDING: <ClockCircleTwoTone twoToneColor="orange" />,
+      STARTED: <LoadingOutlined />,
+      FAILURE: <CloseCircleTwoTone twoToneColor="red" />,
+      MANUAL: <ToolTwoTone twoToneColor="orange" />,
+    };
+    const now = moment();
+    const newJobs = jobs
+      .filter(
+        job =>
+          job.type === "convert_to_wkw" &&
+          job.state !== "SUCCESS" &&
+          moment.duration(now.diff(job.createdAt)).asDays() <= 3,
+      )
+      .sort((a, b) => b.createdAt - a.createdAt);
+
+    const newJobsHeader = (
+      <React.Fragment>
+        Converting datasets{" "}
+        <Tooltip
+          title="The displayed datasets are being converted in the 3 days."
+          placement="right"
+        >
+          <InfoCircleOutlined />
+        </Tooltip>
+      </React.Fragment>
+    );
+    const newJobsList = (
+      <React.Fragment>
+        {newJobs.slice(0, 5).map(job => (
+          <Row key={job.id} gutter={16}>
+            <Col span={10}>
+              <Tooltip title={job.state}>{stateToIcon[job.state]}</Tooltip>
+              {` ${job.datasetName || "UNKNOWN"}`}
```

Comment:
Is there any additional info that could be displayed here? For example, when the job was triggered and/or who triggered it.

Reply:
Thank you for your suggestion. I included the trigger date. All of the jobs that are visible are triggered by yourself, so there is not a benefit of displaying the "owner" of the job

Reply:
are you sure about this? won't admins see all jobs?

Reply:
(I think, you are right; even as a super admin, the jobs page also shows only my jobs)